package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dash-ag-grid",
-    "version": "2.4.0",
+    "version": "31.0.1rc3",
```

Comment:
I'd prefer we not try to match the minor or patch version to AG Grid, just the major. That way we still have the flexibility to add features and fix bugs on our own schedule, we just have to hold breaking changes to align with upgrading the AG Grid major version. So the new release should be `31.0.0`

Reply:
@sdidier-dev - can you chime in here?  This may affect how we link to the correct AG Grid version in the docs

Reply:
Yes, I think we need to be in line about how we'll handle the dag version vs AG Grid version to know how to deal with the links to AG Grid docs: if they go to the last version of AG Grid docs or to an "archive" version of their docs.

Reply:
Actually that shouldn't be an issue for the links!
We took a look with Ann Marie, we're using the version of the ag grid package used in dag:
`AG_grid_version = dag.package['dependencies']['ag-grid-community'].lstrip('^')`
then compare it to the latest version of AG Grid coming from their github repo:
`latest_tag_url = "https://api.github.com/repos/ag-grid/ag-grid/releases/latest"`

So the version of dag can be anything that's should be fine for the links to AG Grid docs 🌈

Reply:
@alexcjohnson,

What are your thoughts on being able to query the underlying grid version from the clientside? Or... could this just be done by searching the source code?

Reply:
It would be handy have a utility -- maybe something like `dag.get_grid_version()`